chapters.md

```diff
@@ -222,4 +78,188 @@ The next 4 following octets correspond to the [User Operation flags](http://dvd.
 
 ChapProcessData contains the pre/post/cell commands in binary format as there are stored on a DVD. There is just an octet preceeding these data to specify the number of commands in the element. As follows: [# of commands(1)][command 1 (8)][command 2 (8)][command 3 (8)].
 
-More information on the DVD commands and format on [DVD-replica](http://www.dvd-replica.com/DVD/), where we got most of the info about it. You can also get information on DVD from [the DVDinfo project](http://dvd.sourceforge.net/dvdinfo/).
\ No newline at end of file
+More information on the DVD commands and format on [DVD-replica](http://www.dvd-replica.com/DVD/), where we got most of the info about it. You can also get information on DVD from [the DVDinfo project](http://dvd.sourceforge.net/dvdinfo/).
+
+## Example 1 : basic chaptering
+
+In this example a movie is split in different chapters. It could also just be an audio file (album) on which each track corresponds to a chapter.
+
+*   00000ms - 05000ms : Intro
+*   05000ms - 25000ms : Before the crime
+*   25000ms - 27500ms : The crime
+*   27500ms - 38000ms : The killer arrested
+*   38000ms - 43000ms : Credits
+
+This would translate in the following matroska form :
+
+```xml
+<Chapters>
+  <EditionEntry>
+    <ChapterAtom>
+      <ChapterUID format="hex">12 34 56</ChapterUID>
```

Comment:
Hmm, ChapterUID is defined as an unsigned integer, so I should change that. I used the practice of `format=hex` from @mbunkus's examples at https://github.com/mbunkus/mkvtoolnix/blob/029a6bf943cc9e4262da80d0851fcca574fceb2e/examples/example-segmentinfo-1.xml; however I neglected that some `UID` Elements are binary and some are unsigned integer.